lang/javascript.py

```diff
@@ -1,120 +1,101 @@
 from talon.voice import Context, Key, press
 import talon.clip as clip
-from ..utils import (
-    text,
-    parse_words,
-    parse_words_as_integer,
-    insert,
-    word,
-    join_words,
-    is_filetype,
-)
+from ..utils import text, parse_words, parse_words_as_integer, insert, word, join_words
 
-JS_EXTENSIONS = (".js", ".jsx")
-
-context = Context("javascript", func=is_filetype(JS_EXTENSIONS))
+def verifyExtension(app, win):
```

Comment:
why create this function? it doesn't appear to be used

Reply:
I took care of this by utilizing the function to scope javascript.py two files with the extension '.js', '.jsx', and 'freeCodeCamp' in the browser. I wasn't making use of earlier it because I was using the file globally.